RFC/src/RFC-0300_DAN.md

```diff
@@ -1 +1,179 @@
+* It is the VNs responsibility to ensure that every state change in a digital asset conforms to the contract's rules.
+* VNs accept digital asset [Instructions] from clients and peers. [Instructions] allow for creating, updating, and
+  expiring digital assets on the DAN.
+* VNs provide additional collateral when accepting an offer to manage an asset, which is stored in a multi-signature
+  contract on the base layer. This collateral can be taken from the VN if it is proven that the VN engaged in
+  malicious behaviour.
+* VNs participate in fraud proof validations in the event of consensus disputes (which could result in the malicious VN's
+  collateral being slashed).
+* Digital asset metadata (e.g. large images) are managed by VNs, but whether the data is considered part of the state
+  (and thus checkpointed) or out of state depends on the type of digital asset contract employed.
+
+#### Checkpoints
+VNs periodically post checkpoint summaries to the [base layer] for each asset that they are managing. The checkpoints will form an immutable
+record of the [Digital Asset] state on the base-layer. There will be two types of checkpoints:
+* An opening checkpoint will:
+  * Specify the members of the VN committee.
+  * Lock up the collateral for the committee members for this checkpoint period.
+  * Collect the fee pool for this checkpoint period from the Asset Issuer into a Multi-Sig UTXO under the control of the committee.
+  This can be a top-up of the fees or a whole new fee pool.
+
+* A closing checkpoint will:
+  * Summarize the Digital Asset state on the base layer.
+  * Release the fee pay outs.
+  * Release the collateral for the committee members for this checkpoint period.
+  * Allow for committee members to resign from the committee
+
+#### Consensus
+Committees of VNs will use a [ConsensusStrategy] to manage the process of
+* Propogating signed instructions between members of the committee.
+* Determining when the threshold has been reached for an instruction to be considered valid.
+
+Part of the [ConsensusStrategy] will be mechanisms for detecting actions by [Bad Actor]s. The nature of the enforcement actions that can be taken
+against bad actors are still to be decided.
+
+#### Fees
+The distribution of the fees for work done by VNs during a checkpoint period will be agreed upon by the committee.
+The committee members will sign for the fee payments based on the respective work done by the committee members. Remaining fees will
+roll over into the next checkpoint periods fee pool.
+
+The exact mechanism for the Multi-Signature address on the base layer and who pays the various fees is still under discussion.
+
+### Network communication
+The VNs will communicate using a peer-to-peer (P2P) network. To facilitate this this VNs must perform the following functions:
+* VNs must maintain a list of peers, and which assets each peer is managing.
+* VNs must relay [instructions] to interested peers.
```

Comment:
Should we define "interested peers" more precisely here? Would it be a superset of the committee ever? If not, maybe just say "committee members" here